annotate.gemspec

```diff
@@ -7,7 +7,7 @@ Gem::Specification.new do |s|
   s.name = "annotate"
   s.version = Annotate.version
 
-  s.required_ruby_version = '>= 1.9.3'
+  s.required_ruby_version = '>= 2.2.5'
```

Comment:
If Ruby 2.1.x exist, `Gem::InstallError` occurs because `ruby_dep requires Ruby version >= 2.2.5, ~> 2.2.`.

<img width="624" alt="2016-08-31 3 06 45" src="https://cloud.githubusercontent.com/assets/8242734/18101000/0bcb5c7a-6f28-11e6-9cfe-cd1e48613049.png">